.github/workflows/docker-ort-runtime-ext.yml

```diff
@@ -20,7 +20,7 @@ name: Docker extended runtime image
 on:
   workflow_dispatch:
   schedule:
-    - cron: '0 4 * * *'
+    - cron: '0 5 * * *'
```

Comment:
This change IMO should be a separate commit. e.g. "align job dispatch time of ort-runtime-ext and ort-runtime"

Reply:
For the commit which re-enables it, IMO it would be nice to have a reference to the commit which accidentally disabled it.

Reply:
@fviernau We never disabled. Actually, we moved from push events to scheduled events.
And after we moved, the schedule event were never handled.

Reply:
@fviernau Splitted in two commits and commented about previous commit. 
The next window to build will be 4 AM and 5 AM.